.circleci/config.yml

```diff
@@ -119,3 +148,10 @@ workflows:
             branches:
               only:
                 - master
+  test262:
+    jobs:
+      - approve-test262-run:
```

Comment:
While `approval` grants contributor rights to run test262 on specific PR, I am wondering if it will end up with many Circle CI builds in the `ON HOLD` status but no one approves them at all.

The good news is that it does not count on job numbers so our new CircleCI job would not be blocked.

Ideally after we merge the PR into master, the regarding test262 job should be suspended if it is still on hold. Is it possible to do so?

Reply:
I _think_ that Circle CI will clean up jobs of merged branches after some time anyway, but I'm not 100% sure.

Reply:
We can merge it and watch.